src/components/common/Layout/Layout.jsx

```diff
@@ -1,11 +1,23 @@
 import React from 'react';
 import Header from '../Header/Header';
+import useAuth from 'hooks/useAuth';
+import Map from 'components/common/Map/Map';
```

Comment:
would you consider to use an index file to simplify this imports? to get something like this: import { Map, PhoneAppSection } from 'components' or import { Map, PhoneAppSection } from 'components/common'

Reply:
I don't like using index files in every folder because it gets really difficult to know in which file you are working if you have more than one index opened.